spec/mnconvert_spec.rb

```diff
@@ -64,6 +64,24 @@
     expect(sts_path.read).to include '<preformat preformat-type="ruby">'
   end
 
+  it "converts XML to IEEE XML", :ci do
+    sts_path = Pathname.new(Dir.pwd) / "rice-en.cd.ieee.xml"
```

Comment:
This test is clearly wrong. We should not convert an Metanorma ISO XML into IEEE XML. We must remove this.